tools/pd-ctl/pdctl/command/region_command.go

```diff
@@ -226,6 +231,46 @@ func showRegionsTopCommand(prefix string) run {
 	}
 }
 
+func showTopReadRegions(cmd *cobra.Command, args []string) {
+	// default to show top read flow
+	if len(args) == 0 {
+		showRegionsTopCommand(regionsReadFlowPrefix)(cmd, args)
+		return
+	}
+	// show top read flow or query
+	if args[0] == "query" || args[0] == "byte" {
+		switch args[0] {
+		case "query":
+			showRegionsTopCommand(regionsReadQueryPrefix)(cmd, args[1:])
+		default: // byte
+			showRegionsTopCommand(regionsReadFlowPrefix)(cmd, args[1:])
+		}
+		return
+	}
+	// default to show top read flow with limit
+	showRegionsTopCommand(regionsReadFlowPrefix)(cmd, args)
+}
+
+func showTopWriteRegions(cmd *cobra.Command, args []string) {
+	// default to show top write flow
+	if len(args) == 0 {
+		showRegionsTopCommand(regionsWriteFlowPrefix)(cmd, args)
+		return
+	}
+	// show top write flow or query
+	if args[0] == "query" || args[0] == "byte" {
+		switch args[0] {
+		case "query":
+			showRegionsTopCommand(regionsWriteQueryPrefix)(cmd, args[1:])
+		default: // byte
+			showRegionsTopCommand(regionsWriteFlowPrefix)(cmd, args[1:])
+		}
+		return
+	}
+	// default to show top write flow with limit
+	showRegionsTopCommand(regionsWriteFlowPrefix)(cmd, args)
```

Comment:
Can we refine to it?
```suggestion
    switch args[0] {
    case "query":
        showRegionsTopCommand(regionsReadQueryPrefix)(cmd, args[1:])
    case "byte":
    	showRegionsTopCommand(regionsWriteFlowPrefix)(cmd, args[1:])
    default:
	showRegionsTopCommand(regionsWriteFlowPrefix)(cmd, args)
    }
```